packages/react-examples/src/react/FocusTrapZone/e2e/FocusTrapZone.e2e.tsx

```diff
@@ -1,50 +1,51 @@
-import type { IFocusTrapZoneProps } from '@fluentui/react/lib/FocusTrapZone';
+import * as React from 'react';
```

Comment:
If you're changing this can you also delete the e2e-specific story files from react-examples? (packages/react-examples/src/react/FocusTrapZone/e2e)

Reply:
And the code in react-examples/.storybook/preview.js to handle loading e2e stories.

Either that, or revert this for now and either you or I can make a separate PR for just the v8 parts after this merges.

Reply:
I can remote the react-examples code, mainly because I would need to find a way to run cypress in 2 modes otherwise 😅

Reply:
@ecraig12345  I've removed the entire `e2e` folder for FTZ in react-examples